frontend/app/src/components/home/CurrentChatMessages.svelte

```diff
@@ -496,18 +497,18 @@
         return firstKey;
     }
 
-    function chatUpdated(): void {
```

Comment:
On chatUpdated we now check for earlier and later events, because for a new chat you can be within the loading distance of the top and the bottom.